src/main/java/seedu/finance/logic/parser/ParserUtil.java

```diff
@@ -90,6 +90,9 @@ public static Date parseDate(String date) throws ParseException {
     public static Path parseFile(String filename) throws ParseException {
         requireNonNull(filename);
         String trimmedFilename = filename.trim();
+        if (trimmedFilename.indexOf('/') != -1) {
+            throw new ParseException("Filename should not contain '\\' character.");
```

Comment:
File name should not contain other special characters as well such as `*`, `?`, `,`, `(`, `)`, `~`